analyses/cms-open-data-ttbar/utils/file_input.py

```diff
@@ -119,43 +110,3 @@ def download_file(url, out_file):
     out_path.parent.mkdir(parents=True, exist_ok=True)
     with tqdm.tqdm(unit='B', unit_scale=True, unit_divisor=1024, miniters=1, desc=out_path.name) as t:
         urllib.request.urlretrieve(url, out_path.absolute(), reporthook=tqdm_urlretrieve_hook(t))
-
-
-class ServiceXDatasetGroup():
```

Comment:
`numpy` is no longer needed here with `ServiceXDatasetGroup` removed (this is about line 2 but GitHub won't let me comment there as it's not part of the diff)